### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by MinimalStructure**
I added a Trezor Wallet explanation and an Option C: Blue Wallet for the Emergency Recovery

### CODE REVIEW

File: _pages/emergency-recovery/index.md [hunk 1]
```diff
@@ -24,7 +24,113 @@ As there are "gotchas" in both of the below setups, we recommend you read [the a
...
+
+Click wipe device
+Confirm you want to wipe your device. I am assuming there are no funds on this wallet.
+Confirm the wipe on your device.
+Wipe is completed, disconnect your device.
+Plug it back in:
+
+5. Click “Recover an existing wallet”. You are recovering 24 words, and you want to used advanced recovery because you NEVER TYPE OUR SEED WORDS INTO YOUB BROWSERS. You also did not use a passphrase with our seed picker wallet, so you will leave that unchecked. Click Continue. Agree to the terms of service in your device.
+
+7. To enter our seed into the device, you will navigate the scrambled BIP39 wordlist on our Trezor.  In this example I will enter “zoo” 23 times and “buddy” one time. The screen says, “Select letters for the 1st word” with 9 squares with dots bellow it. Each square with a dot in the middle represents one of the 9 digits on your keypad.  The top row represents 7, 8, and 9.  The middle row represents 4,5, and 6.  The bottom row represents 1, 2, and 3 on your keypad.
```
Comment (mflaxman) [inline _pages/emergency-recovery/index.md:72]: You skipped `6.`, but actually it's easiest/best to make everything `1.` and jekyll will auto-increment.

File: _pages/emergency-recovery/index.md [hunk 2]
```diff
@@ -24,7 +24,113 @@ As there are "gotchas" in both of the below setups, we recommend you read [the a
...
+
+Our device also says, “Please enter the 1st word of you mnemonic” and it gives us groups of scrambled letters.  These letters correspond with the 9 keys on our number pad.  In this example:
```
Comment (mflaxman) [inline _pages/emergency-recovery/index.md:74]: This is pretty confusing, perhaps we should (for now) only recommend the Model T. It's still stateless, but it at least has a "keyboard" on the touchscreen which doesn't require a substitution cipher.

File: _pages/emergency-recovery/index.md [hunk 3]
```diff
@@ -41,7 +147,26 @@ Electrum will display these as `xpub` master keys.
...
-1. You can then save this PSBT file and broadcast it via Specter-Desktop. 
+If correctly imported, Electrum will then detect that this transaction has been signed by 1 of 2 required signatures, and present you the opportunity to sign it with the seedpicker paper wallet key.
+1. You can then save this PSBT file and broadcast it via Specter-Desktop.
 
 {% include next_steps.md %}
+
+#### Option C BlueWallet
+1. TODO Add screen shots and a video
+2.
+3. Blue Wallet is an open-source wallet available on the App Store, Google Play, APK and Desktop for macOS. It is free, readily available and easy to use. **As it will be touching private key material, we strongly recommend you run this on an airgapped machine. An old phone on Airplane Mode oworks well for this**
```
Comment (mflaxman) [inline _pages/emergency-recovery/index.md:158]: Misspelled works

File: _pages/emergency-recovery/index.md [hunk 4]
```diff
@@ -41,7 +147,26 @@ Electrum will display these as `xpub` master keys.
...
+4.
+5. 1. Download Blue Wallet from the Appstore, Google Play or the [bluewallet.io](https://bluewallet.io/).
```
Comment (mflaxman) [inline _pages/emergency-recovery/index.md:160]: What's with the numbering here?

File: _pages/emergency-recovery/index.md [hunk 5]
```diff
@@ -41,7 +147,26 @@ Electrum will display these as `xpub` master keys.
...
 1. You can generate a PSBT (Partially Signed Bitcoin Transaction) from Specter-Desktop, sign it with one of your keys (e.g. the Coldcard or Cobo), and then load that PSBT into Electrum via `Tools` > `Load from file`.
-If correctly imported, Electrum will then detect that this transaction has been signed by 1 of 2 required signatures, and present you the opportunity to sign it with the seedpicker paper wallet key. 
-1. You can then save this PSBT file and broadcast it via Specter-Desktop. 
+If correctly imported, Electrum will then detect that this transaction has been signed by 1 of 2 required signatures, and present you the opportunity to sign it with the seedpicker paper wallet key.
+1. You can then save this PSBT file and broadcast it via Specter-Desktop.
 
 {% include next_steps.md %}
+
+#### Option C BlueWallet
+1. TODO Add screen shots and a video
```
Comment (mflaxman) [inline _pages/emergency-recovery/index.md:156]: Is there a link to a video/tutorial anywhere?

**Review by mflaxman — CHANGES_REQUESTED** [review]
Thanks for the PR! I like the general direction, but I don't think we should recommend the Trezor One. Advanced users can probably figure it out, but given that it's already stateless (!) the substitution cipher is non-ideal. And the model T has an SD car that's not used for signing, but if they add support for it in the future it could make the Trezor less bad at multisig.

Also, the spellchecker is failing. Please fix misspellings and update `wordlist.txt` to add words to the spellchecker.

**Comment by holgern** [comment]
There must be no newline inbetween a list, otherwise pandoc will not handle them correctly. Please remove all line breaks such that it looks like:
```
1. ...
1. ...
1. ...
```

**Comment by MinimalStructure** [comment]
Back to the drawing board :)

I noticed Blue Wallet only let's users import keys while connected to the Internet.

That's a bad idea.

It could work in an extreme emergency, but  it's not good enough for the guide.

**Comment by mflaxman** [comment]
Oh bummer, unfortunate they don't support an airgap :(

Closing for now, please reopen/edit if you disagree.

**Pull request closed without merging**